### PR TITLE
Release 1.1.1

### PR DESCRIPTION
**Pull request by julianbrost**
Includes version bump, changelog and upgrading docs.

Remaining other PRs to be merged for the release:
* #631: already included in the changelog.
* #634: GitHub Actions only, not relevant for changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+* Fix a possible crash when the Icinga 2 heartbeat is lost. #559
+* Retry additional non-fatal database errors. #593 #583
+* Make heartbeat compatible with Percona XtraDB Cluster. #609
+* Write a hint for empty arrays/dicts into `customvar_flat` for Icinga DB Web. #601
+* Schema: Allow longer names for notification objects. #584
+* Schema: Add missing indices to `hostgroup`, `servicegroup`, and `customvar_flat`. #616 #617
+* Schema: Change sort order of history event types. #626
+* `icingadb-migrate`: Ignore events that miss crucial information. #551
+* `icingadb-migrate`: Fix a foreign key error for flapping history with `ido.from` set. #554
+* `icingadb-migrate`: Fix a constraint violation for flexible downtimes that never started. #623
```
Comment (Al2Klimov) [inline CHANGELOG.md:17]: A clear subsection candidate IMAO.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* Config: Warn about unknown options (`icingadb-migrate` rejects them). #605 #631
```
Comment (Al2Klimov) [inline CHANGELOG.md:18]: Apropos, that could even include this one, but just specifically for the migration.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+This release fixes a few crashes in the Icinga DB daemon, addresses some shortcomings in the database schema,
+and makes the `icingadb-migrate` tool handle malformed events and other edge-cases more reliably.
+
+* Fix a possible crash when the Icinga 2 heartbeat is lost. #559
+* Retry additional non-fatal database errors. #593 #583
+* Make heartbeat compatible with Percona XtraDB Cluster. #609
+* Write a hint for empty arrays/dicts into `customvar_flat` for Icinga DB Web. #601
+* Schema: Allow longer names for notification objects. #584
+* Schema: Add missing indices to `hostgroup`, `servicegroup`, and `customvar_flat`. #616 #617
+* Schema: Change sort order of history event types. #626
```
Comment (Al2Klimov) [inline CHANGELOG.md:14]: Also a subsection candidate. (Schema-only things.)

File: CHANGELOG.md (hunk 1 above)
Comment (julianbrost) [reply CHANGELOG.md]: Thought about sections but apart from schema and icingadb-migrate, there isn't much to group and having most changes in a misc section is also somewhat strange. Anyways, let me know what you think about what I've just pushed.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**